Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apk add --update --no-cache tzdata && \
     echo "Europe/London" > /etc/timezone
 
 RUN apk add --update --no-cache --virtual runtime-dependances \
-    yarn shared-mime-info
+    yarn shared-mime-info git
```

Comment:
What is this needed in the Docker container for? I can't see where git is used there ...

Reply:
Because we're pulling from GitHub until we get RubyGems

Reply:
Ah, for when we run `bundle` in the Docker container. Of course. 👍